e2e-tests/pages/dashboardPage.ts

```diff
@@ -108,7 +110,7 @@ export class DashboardPage {
 
         // generate a new mask and confirm
         await this.generateNewMaskButton.click()
-        await this.page.waitForSelector(this.maskCard, { timeout: 3000 })
+        await this.page.waitForSelector(this.maskCardString, { timeout: 3000 })
 
         // randomize between 1.5-2.5 secs between each generate to deal with issue of multiple quick clicks
```

Comment:
irrelevant comment: i think `await this.page.waitForTimeout((Math.random() * 2500) + 1500)` is 1.5s - 1.5+2.5s (so 1.5-4 secs)